addons/download-button/style.css

```diff
@@ -0,0 +1,49 @@
+#sa-download-button:before {
+  background-image: url("./download.svg");
+  display: inline-block;
+  margin-top: -2px;
+  margin-right: 0.25rem;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: contain;
+  width: 1.125rem;
+  height: 1.125rem;
+  vertical-align: middle;
+  content: "";
+}
+
+#sa-download-button.waiting {
+  opacity: 0.6;
+  pointer-events: none;
+}
+
+@keyframes spin-intro {
+  0% {
+    transform: scale(0);
+    opacity: 0;
+  }
+  100% {
+    transform: scale(1);
+    opacity: 1;
+  }
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0);
+  }
+  100% {
+    transform: rotate(359deg);
+  }
+}
+
+#sa-download-button.loading:before {
```

Comment:
```suggestion
#sa-download-button.loading:before,
#sa-download-button.waiting:before {
```

Reply:
Displaying a spinner on the button during a download is fine, but if we show the spinner while the project is loading, it might make it seem like the addon is doing something.